runtime/nls/j9gc/j9modron.nls

```diff
@@ -989,3 +989,22 @@ J9NLS_GC_OPTIONS_HIERARCHICAL_SCAN_ORDERING_NOT_SUPPORTED_WARN.system_action=The
 J9NLS_GC_OPTIONS_HIERARCHICAL_SCAN_ORDERING_NOT_SUPPORTED_WARN.user_response=Refer to the OpenJ9 documentation for -Xgc:hierarchicalScanOrdering.
 
 # END NON-TRANSLATABLE
+
+J9NLS_CHECKPOINTGCTHREAD_VALUE_MUST_BE_AT_MOST_SPECIFIED_GCTHREAD_VALUE_WARN=-XX:CheckpointGCThreads will be ignored. The specified value %zu must not be larger than user specified value of the total GC threads %zu using -Xgcthreads, -Xgcmaxthreads or -Xgc:threads
+# START NON-TRANSLATABLE
+J9NLS_CHECKPOINTGCTHREAD_VALUE_MUST_BE_AT_MOST_SPECIFIED_GCTHREAD_VALUE_WARN.explanation=-XX:CheckpointGCThreads must be smaller than or equal to the number of gc threads
+J9NLS_CHECKPOINTGCTHREAD_VALUE_MUST_BE_AT_MOST_SPECIFIED_GCTHREAD_VALUE_WARN.system_action=The JVM ignores the -XX:CheckpointGCThreads option.
+J9NLS_CHECKPOINTGCTHREAD_VALUE_MUST_BE_AT_MOST_SPECIFIED_GCTHREAD_VALUE_WARN.user_response=Refer to the OpenJ9 documentation and adjust -Xgcthreads/-XX:CheckpointGCThreads to make sure number of gc threads is larger than or equal to the number of checkpointgcthreads
```

Comment:
The sentance should end with a period. Similarly for J9NLS_CHECKPOINTGCTHREAD_VALUE_MUST_BE_AT_MOST_HEURISTIC_GCTHREAD_VALUE_WARN

Reply:
I see many of the messages in this file don't use periods, but some do. These new ones could be consistent, in that all of explanation, system_action, user_response should all either have one, or not.